src/assets/scss/Login.scss

```diff
@@ -128,3 +129,30 @@ body {
   font-size: rem(16);
   color: #f00
 }
+
+.innerSocial {
+  display: flex;
+  align-items: center;
+  margin-top: 0.625rem;
+}
+
+
+
+.content-continue {
+  padding:0px;
+  margin:0px;
+}
+
+.google-login {
+  margin-top:20px;
+}
+
+.facebookbutton{
+  padding: 0;
+  margin-right: 15px;
+  display: block;
+}
+.google-login{
+  align-content: center;
+  margin-top: 23px;
```

Comment:
Hey @dannylwe,
please convert px to rem

Reply:
@ezrogha worked on feedback